locations/spiders/mcdonalds_us.py

```diff
@@ -8,13 +8,14 @@ class McDonaldsUSSpider(scrapy.Spider):
     name = "mcdonalds_us"
     item_attributes = {"brand": "McDonald's", "brand_wikidata": "Q38076"}
     allowed_domains = ["www.mcdonalds.com"]
+    download_delay = 0.2
 
     def start_requests(self):
         # Experimentally found that 250 was max results
-        base_url = "https://www.mcdonalds.com/googleappsv2/geolocation?latitude={lat}&longitude={lng}&radius=100&maxResults=250&country=us&language=en-us"
+        base_url = "https://www.mcdonalds.com/googleappsv2/geolocation?latitude={lat}&longitude={lng}&radius=50&maxResults=250&country=us&language=en-us"
```

Comment:
When I test this in New York, it seems to fail with a blank response when maxResults is 250. It works with 200, so we might have to stop there. That's still not enough for the most dense areas, though.

in the future, we might add a recursive call with a smaller radius and a slightly offset lat/lon to get the rest?

Reply:
Scratch that, I seem to get more then 213 features with 25 mile radius (maxResults seems to max out there for that size) in NYC. 

10 miles still accepts the parameter to 250 (but I think it might fail if it actually returns that value) and only returns 89 for the given lat/long